app/Platform/Services/GameTopAchieversService.php

```diff
@@ -178,7 +178,8 @@ private function convertPlayerGames(Collection $playerGames): array
 
             $retval[] = [
                 'user_id' => $playerGame->user_id,
-                'user' => $playerGame->user,
+                'user_display_name' => $playerGame->user->display_name,
+                'user_avatar_url' => $playerGame->user->avatar_url,
```

Comment:
I can't access any game pages now. Both 'lastest-masters' and 'most-points' files are still referencing `user`.

Reply:
Fixed in latest with a few `whereIn()` queries in those components to avoid an N+1 query problem.